gitops-wg/README.md

```diff
@@ -0,0 +1,59 @@
+<p><img src="https://raw.githubusercontent.com/gitops-working-group/gitops-working-group/main/assets/logos/GWG/icon/color/gitopswg-icon-color.svg" alt="GitOps Working Group logo icon color" width="200"></p>
+
+The GitOps WG focus is to clearly define a vendor-neutral, principle-led meaning of GitOps, which will establish a foundation for interoperability between tools, conformance, and certification.
+Lasting GitOps programs, documents, and code will live within the CNCF [OpenGitOps](https://github.com/open-gitops) project, also guided by the WG.
+
+See the WG charter here: <https://github.com/cncf/tag-app-delivery/blob/master/gitops-wg/charter.md>
+
+## Principles
+
+The GitOps Principles aim to give participants in the cloud native ecosystem a common understanding of GitOps systems based on shared principles, rather than individual opinion or implementation preference, as the foundation for GitOps practices.
+
+See releases here: <https://github.com/open-gitops/documents/releases>
+
+## Community
+
+- Slack: `#wg-gitops` in [CNCF workspace](https://slack.cncf.io/)
+- Mailing list: <https://lists.cncf.io/g/cncf-tag-app-delivery/topics>. Watch or participate in topics prefixed with `[gitops-wg]`
+- Attend a [Working Group or Committee meeting](#meetings)
+- Volunteer: <https://github.com/open-gitops/project/blob/main/GOVERNANCE.md#committees>. Help with projects according to your interest and ability
+- GitOps Subreddit: <https://www.reddit.com/r/GitOps/>
+- Twitter: <https://twitter.com/gitopswg>
+- Express your interest: [interested-parties.md](./interested-parties.md)
+- Follow OpenGitOps: <https://github.com/open-gitops/project>
+
+## Meetings
+
+- 🗓 The WG now meets every other Wednesday at [18:00 GMT](https://greenwichmeantime.com/time-gadgets/time-zone-converter/)
+- 👩‍💻 Join using [this Zoom link](https://zoom.us/j/93779536510?pwd=TEFzbGRzREI3MVlkZmluemVkMEhHdz09)
+- ✏️ See [Agenda and Notes](https://docs.google.com/document/d/1hxifmCdOV5_FbKloDJRWZQHq0ge-trXJKF-BgV4wHVk/) for next and and all previous meetings
+- 💡 Hint: use the [document outline](https://support.google.com/docs/answer/6367684) to help navigate to the right meeting!
+- 📼 Browse the [YouTube playlists](https://www.youtube.com/channel/UCI6iqYuuI4gZuOCZaks5i1g/playlists) for recordings of all previous meetings
+- 📧 Additional committee meetings may be scheduled as needed, and will be announced on the [CNCF App Delivery TAG](https://github.com/cncf/tag-app-delivery) mailing list, prefixed with `[gitops-wg]`
+
+## Timeline
+
+This timeline is a work in progress.
+If you are interested in keeping abreast of any new developments, please see [How to Get Involved](https://opengitops.dev/get-involved/).
+
+| When                      | What                                                                                                                                                                                                                          |
```

Comment:
@lloydchang my personal pref is to keep the older table style, to more easily see content diffs when updating the table. When aligning like this whenever there's a longer line, the entire table needs to be updated rather than just appending as in the earlier style.

Reply:
@scottrigby asked:
> Question: do you know why the git diff is not showing, and it appears to be an entirely new file? Are there perhaps line-ending changes?

Why: I renamed the file from `README.MD` to `README.md` to align with [other `README.md` files throughout this repository](https://github.com/cncf/tag-app-delivery/blob/master/README.md), and with [the `README.md` file in the gitops-working-group repository](https://github.com/gitops-working-group/gitops-working-group/blob/main/README.md).

What:  I want to mitigate the minor risk of other people encountering the Developer Experience (DX) with `git add` and `git status` that I experienced.

How: The DX has been:
1. `git add README.md`
2. `git add` failed silently with a `0` exit code
3. `git status` won't display `README.md` because the filename was previously named `README.MD`

Where: I don't know if the previous filename `README.MD` from https://github.com/cncf/tag-app-delivery/pull/146/files was created by a sticky shift key, or sticky caps lock key ⌨️, or by a typo 😅

As an inside joke about this DX — If you don't mind, then I can start calling you by the nicknames of Doc or Dr. @scottrigby, M.D. ... On second thought, I'm just kidding about calling you by nicknames 😄

Reply:
Oh ha that was prob me. Let's not mix renaming the file into the same PR though, at least not like this, because it makes it nearly impossible to see the diff of what you're actually proposing changing